src/test/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/AlkylStructureFragmenterTest.java

```diff
@@ -0,0 +1,754 @@
+     * @throws FileNotFoundException if test structures file can not be found or accessed
+     * @throws URISyntaxException if syntax of path to test structures file is not correct
+     */
+    @Test
+    public void linearDoubleBondExtractionTest() throws CloneNotSupportedException, FileNotFoundException, URISyntaxException {
+        IAtomContainerSet tmpReadAtomContainerSet = this.readStructuresToACSet("de.unijena.cheminf.mortar.model.fragmentation.algorithm.ASF/ASF_Butene_Test.mol");
+        IAtomContainer tmpButeneAC = tmpReadAtomContainerSet.getAtomContainer(0);
+        AlkylStructureFragmenter tmpASF = this.getDefaultASFInstance(tmpButeneAC, false, false, true);
+        List<IAtomContainer> tmpFragmentACList = tmpASF.fragmentMolecule(tmpButeneAC);
+        List<String> tmpExpectedList = new ArrayList<>(3);
+        tmpExpectedList.add("C");
+        tmpExpectedList.add("C");
+        tmpExpectedList.add("C=C");
+        List<String> tmpFragmentStringList = new ArrayList<>();
+        for (IAtomContainer tmpAC: tmpFragmentACList) {
+            tmpFragmentStringList.add(ChemUtil.createUniqueSmiles(tmpAC, false));
+        }
+        Assertions.assertTrue(this.compareListsIgnoringOrder((ArrayList) tmpExpectedList, (ArrayList) tmpFragmentStringList));
+    }
+    /**
+     * Method to test a default alkyl structure fragmentation on a concept molecule covering a broad range of resulting fragments.
+     *
+     * @throws Exception if fragmentation does not result in expected fragments
+     */
+    @Test
+    public void defaultFragmentationTest() throws Exception {
+        SmilesParser tmpParser = new SmilesParser(SilentChemObjectBuilder.getInstance());
+        //test structure: C12=CC(=CC=C1C(=CC(=C2)C3CCCCC3)CC(C)(C)C/C=C/CCC)/C=C\C(C)C
+        IAtomContainer tmpTestStructureAC = tmpParser.parseSmiles("C12=CC(=CC=C1C(=CC(=C2)C3CCCCC3)CC(C)(C)C/C=C/CCC)/C=C\\C(C)C");
+        AlkylStructureFragmenter tmpASF = this.getDefaultASFInstance(tmpTestStructureAC, false, false, true);
+        List<String> tmpResultSMILESList = this.generateSMILESFromACList(tmpASF.fragmentMolecule(tmpTestStructureAC));
+        List<String> tmpExpectedSMILESList = new ArrayList<>(5);
+        tmpExpectedSMILESList.add("C=1C=CC2=CC(=CC=C2C1)C=CC(C)C");
+        tmpExpectedSMILESList.add("CC(C)(C)C");
+        tmpExpectedSMILESList.add("C=C");
+        tmpExpectedSMILESList.add("CCC");
+        tmpExpectedSMILESList.add("C1CCCCC1");
+        Assertions.assertTrue(this.compareListsIgnoringOrder(new ArrayList<>(tmpResultSMILESList),
+                new ArrayList<>(tmpExpectedSMILESList)));
+    }
+
+    /**
+     * Example fragmentation using natural compounds CNP0321029.1; CNP0509407.0 and CNP0242949.0. The natural compounds
+     * used in this test can be found in the natural product database COCONUT @see <a href="https://coconut.naturalproducts.net/"<a/>.
```

Comment:
this doesn't render correctly, 'at'see is a tag that cannot be used in this way.